h/app.py

```diff
@@ -29,6 +29,9 @@ def create_app(settings):
         api_view = wsgiapp2(api_app)
         config.add_view(api_view, name='api')
 
+    if config.registry.feature('claim'):
```

Comment:
Oh, and IMO all major new chunks of code should be behind a feature flag, so that we can merge self-consistent, well tested, but possibly incomplete work to master, without having to hold up deployments to prod until the feature is complete.